src/util.ts

```diff
@@ -9,7 +9,7 @@ import * as fs from 'fs-extra';
 import * as yaml from 'yaml';
 import { ValidationConfig } from './config';
 import { PluginManager } from './plugins/_manager';
-import { ValidationPlugin, ValidationContext, ValidationReport, Validation } from './plugins/validation';
+import { Validation, ValidationContext, ValidationPlugin, ValidationReport } from './plugins/validation';
```

Comment:
How come the order here changes? Do you have a different VSCode setting or something?

Reply:
Yeah, looks like this is a linter thing that sorts imports. Will fix it.